config/maintainers.json

```
@@ -11,6 +11,16 @@
       "avatar_url": null,
       "bio": "I have always been interested in Lisps since taking the SICP course in college. I love the expressiveness and conciseness of the simple syntax, and the ability to build the language upward to meet the domain."
     },
+    {
+      "github_username": "thelostlambda",
+      "alumnus": false,
+      "show_on_website": false,
+      "name": "Brooks J Rady",
+      "link_text": null,
+      "link_url": "thelostlambda.xyz",
```

Could you add the `http://` prefix?

```suggestion
      "link_url": "http://thelostlambda.xyz",
```